core/misc/dropbutton.css

```diff
@@ -5,6 +5,14 @@
  */
 
 /**
+ * Set width of dropbuttons where extra width is needed
+ */
+#system-modules .dropbutton-wrapper div,
+#layout-edit .dropbutton-multiple div,
+#layout-list .dropbutton-wrapper div {
+  min-width: 140px;
+}
```

Comment:
The dropbutton.css file shouldn't contain any module-specific selectors like this. These should be moved to `system.admin.css` and `layout.admin.css` respectively.